.npmrc

```diff
@@ -0,0 +1,4 @@
+_auth = ${NPM_AUTH}
+always-auth = true
+strict-ssl = true
+registry = https://npm.goodeggs.com/
```

Comment:
Looks like you followed "How to configure a private module or app" rather than "How to configure a public module"! This is a public module, so it should be reading from and publishing to the public NPM registry.